.circleci/config.yml

```diff
@@ -101,6 +105,56 @@ node-plugin-base: &node-plugin-base
     - store_artifacts:
         path: ./coverage/lcov-report
 
+node-aws-sdk-base: &node-aws-sdk-base
```

Comment:
This shouldn't need to use a machine executor as `localstack` and `serverless_localstack` can be used as secondary containers.

Reply:
✅